### DISCUSSION TITLE
Vuejs com Spring MVC

### DISCUSSION THREAD

**Discussion by dsoldera** [frontendbr/forum#337] [Perguntas e Respostas]
Olá Pessoal. 

Vcs acham uma boa ideia usar o Vue.js como gerenciador das funções Javascript unicamente, num Projeto usando Springa MVC (Java)? Estou atras de uma ferramenta que me ajuda a otimizar hoje os components criados no Padrão Prototype que seja mais facil de entender/gerenciar e desenvolver. 

Se tiverem um ideia de uma outra ferramenta que trabalha bem com o Spring MVC, aceito sugestões.

**Comment by felipefialho** [comment]
c/c @wesleysaraujo

**Comment by wesleysaraujo** [comment]
@dsoldera Se a ideia é organizar e componentizar suas funções javascript então o Vuejs é uma ótima opção pois ele vai te dar uma estrutura de organização e código bem sólida e totalmente reutilizável. Não conheço a estrutura do Spring MVC, mas acredito que Vuejs se adeque a qualquer framework, já usei ele em várias estruturas de aplicações e funcionou muito bem, tanto usando a estrutura do vue-cli quanto usando ele standalone com gulp, babel, browserify e vuefy.
